Luke Smith Portfolio.css

```diff
@@ -0,0 +1,111 @@
+/** {
+	box-sizing: border-box;
+}
+*/
+body {
+  margin: auto;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.topnav {
+  overflow: auto;
+  background-color: #ffffff;
+}
+
+.topnav a {
+  float: left;
+  color: #000000;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+  width: auto;
+}
+
+.topnav a:hover {
+  background-color: #dddddd;
+  color: #000000;
+}
+
+body {
+  background: #555555;
+}
```

Comment:
You've declared styling for body three times in this document. You should really only have one body {} styling - I would recommend that you put all the styling applied to body in the same entry.